test/user.test.js

```diff
@@ -91,7 +91,7 @@ describe('User', function() {
         assert.equal(err.details.context, User.modelName);
         assert.deepEqual(err.details.codes.email, [
           'presence',
-          'format.null'
+          'custom'
```

Comment:
Could you please improve your patch to report a unique error code, e.g. `isemail`? The validation codes are typically used by front-end to display a localised error message. The code `custom` can mean anything...